project/game_schema.json

```diff
@@ -0,0 +1,296 @@
+            "index": true,
+            "keywords": []
+          },
+          {
+            "name": "hasTags",
+            "description": "The tags attached to this video",
+            "dataType": [
+              "Tag"
+            ],
+            "cardinality": "many",
+            "vectorizePropertyName": false,
+            "index": false
+          },
+          {
+            "name": "duration",
+            "description": "The duration of this video in second",
+            "dataType": [
+              "int"
+            ],
+            "cardinality": "atMostOne",
+            "vectorizePropertyName": false,
+            "index": false,
+            "keywords": []
+          },
+          {
+            "name": "viewCount",
+            "description": "The viewer count of this video",
+            "dataType": [
+              "int"
+            ],
+            "cardinality": "atMostOne",
+            "vectorizePropertyName": false,
+            "index": false,
+            "keywords": []
+          },
+          {
+            "name": "hasSubs",
+            "description": "All subtitles of this video",
+            "dataType": [
+              "Subtitle"
+            ],
+            "cardinality": "many",
+            "vectorizePropertyName": false,
+            "index": false
+          }
+        ]
+      },
+      {
+        "class": "Tag",
+        "description": "A label attached to a Game for the purpose of identification or to give other information",
+        "vectorizeClassName": false,
+        "keywords": [],
+        "properties": [
+          {
+            "name": "name",
+            "description": "The name of this tag",
+            "dataType": [
+              "text"
+            ],
+            "cardinality": "atMostOne",
+            "vectorizePropertyName": true,
+            "index": true,
+            "keywords": []
+          },
+          {
+            "name": "hasGames",
+            "description": "All games within this tag",
+            "dataType": [
+              "Game"
+            ],
+            "cardinality": "many",
+            "vectorizePropertyName": false,
+            "index": false
+          }
+        ]
+      },
+      {
+        "class": "Subtitle",
+        "description": "A caption displayed at the bottom of a video that translate or transcribe the dialogue or narrative",
+        "vectorizeClassName": false,
+        "keywords": [],
+        "properties": [
+          {
+            "name": "text",
+            "description": "The text of this caption",
+            "dataType": [
+              "text"
+            ],
+            "cardinality": "atMostOne",
+            "vectorizePropertyName": true,
+            "index": true,
+            "keywords": []
+          },
+          {
+            "name": "startTime",
+            "description": "The start time of this caption appearance on the video",
+            "dataType": [
+              "text"
```

Comment:
Maybe use Date or String

Reply:
Thanks!
I will take `string` for now.


As for `Date`, An example of `startTime` and `endTime` are something like below, taken from Youtube's subtitle file:
`startTime`: 00:00:06.769
`endTime`: 00:00:06.779

can `Date` be made as time-stamp only instead of datetime-stamp? Because what I got from the doc was something like this:

> Date Type
> Weaviate accepts dates formatted in following the ISO 8601 standard.
> For example: 2019-08-24T14:25:57Z.

When in Django, its kinda like `model.TimeField` instead of `model.DateTimeField`.

Let me know where I'm missing.

Reply:
> can Date be made as time-stamp only instead of datetime-stamp?

Good point, the answer is no. Maybe it is better to change the `startTime` and `endTime` to integers and convert the number during import. For example, 2 minutes and 20 seconds becomes the integer `140`.

Reply:
Thanks for the answer!

Considering the millisecond, let me pull up the scale to 1000, if I may.
So for example, 2 minutes 20 seconds 980 milliseconds becomes the integer `140980` instead.